mux.go

```diff
@@ -27,7 +26,10 @@ func (c *CommandMux) Handle(ctx context.Context, rw *ResponseWriter, cmd io.Read
 
 	_, err := doc.ReadFrom(cmd)
 	if err != nil {
-		c.Logger.Infof("could not read command, err: %s", err)
+		slog.ErrorContext(ctx, "could not read command",
```

Comment:
I don't think it's a good idea to switch this to an error log from info. This would create an error log every time we can't read the input data - which when I look in our logs happens more than one would think.